manubot/cite/doi.py

```diff
@@ -210,7 +231,7 @@ def get_doi_csl_item(doi: str):
 
 
 doi_retrievers = [
```

Comment:
We could reiterate with a comment here why `get_doi_csl_item_datacite` is not in the list.

Reply:
done in https://github.com/manubot/manubot/pull/319/commits/813fcd6dfc7b87f836ebf29b89e2705b1596b5bd